lua/zen-mode/view.lua

```diff
@@ -74,6 +74,7 @@ end
 
 function M.open(opts)
   if not M.is_open() then
+    config.colors(opts)
```

Comment:
this should move to the `M.create` function, right under the `opts = `, to make sure the opts object has all the options.

Your code would give an error with `toggle({})` for example.

Reply:
Oh, how have I not tested that? I'll fix it right now.